core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -191,6 +227,100 @@ oppia.controller('LearnerDashboard', [
       return value;
     };
 
+    $scope.sortFeedbackFunction = function(entity) {
+      return entity[$scope.currentFeedbackThreadsSortType];
+    }
+
+    $scope.onClickThread = function(
+      explorationId, threadId, explorationTitle) {
+      var threadDataUrl = (
+        '/learnerdashboardthreadhandler/' + explorationId + '/' + threadId);
+      $scope.threadTitle = explorationTitle;
```

Comment:
It seems weird to me that the threadTitle is the explorationTitle. If you want to display the latter, why not just call it explorationTitle?